### PR TITLE
added module of pacman and indentation improvement

### PR DESCRIPTION
**Pull request by meets2tarun**
Search module for pacman to apt 
And indentation improvement

### CODE REVIEW

File: ubuntu/yum [hunk 1]
```diff
@@ -10,13 +10,13 @@ case "${array[0]}" in
...
-          case "${array[1]}" in
-          "remove") echo command translated to --> apt remove ${array[2]}
-          apt remove ${array[2]}
-          ;;
-          *)echo "send a pull request to https://github.com/fossasia/mew or make it an issue if its not working"
-          ;;
-          esac
+case "${array[1]}" in
+  "remove") echo command translated to --> apt remove ${array[2]}
+  apt remove ${array[2]}
```
Comment (xeon-zolt) [inline ubuntu/yum]: @meets2tarun explain this part

Comment (meets2tarun) [inline ubuntu/yum]: Nothing changed, I just made it look in a formatted way/

Comment (xeon-zolt) [inline ubuntu/yum]: lets merge it when its complete

Comment (meets2tarun) [inline ubuntu/yum]: I have completed, will soon make a PR.

**Comment by meets2tarun** [comment]
@xeon-zolt Please review and tell me if there is something I left.

**Comment by mariobehling** [comment]
Please help to resolve merge conflicts.

**Pull request merged**